packages/devtools_app/lib/src/app.dart

```diff
@@ -516,6 +518,55 @@ class SettingsDialog extends StatelessWidget {
             toggle: preferences.toggleVmDeveloperMode,
             gaItem: gac.vmDeveloperMode,
           ),
+          Column(
+            children: [
+              Row(
+                children: [
+                  CheckboxSetting(
+                    label: const Text(
+                      'Enable verbose logging',
+                    ),
+                    listenable: preferences.verboseLoggingEnabled,
+                    toggle: (enable) => preferences.setVerboseLogging(enable),
+                    gaItem: gac.verboseLogging,
+                  ),
+                  const SizedBox(
+                    width: defaultSpacing,
+                  ),
+                  IconLabelButton(
+                    label: 'Copy Logs',
+                    icon: Icons.copy,
+                    onPressed: () async => copyToClipboard(
+                      LogStorage.root.toString(),
+                      'Successfully Copied Logs',
+                    ),
+                  ),
+                  const SizedBox(
+                    width: defaultSpacing,
+                  ),
+                  ClearButton(
+                    label: 'Clear Logs',
+                    onPressed: () => LogStorage.root.clear(),
+                  ),
+                ],
+              ),
+              const Row(
+                children: [
+                  SizedBox(
```

Comment:
nit: if this is intended to give the parent row leading padding, just wrap the Row in a Padding widget to make that more clear instead of adding a spacer in the beginning of the Row